scmrepo/git/__init__.py

```diff
@@ -71,6 +77,10 @@ def reset_all(self) -> None:
         for backend in self.initialized.values():
             backend._reset()  # pylint: disable=protected-access
 
+    def move_to_end(self, key: str, last: bool = True):
```

Comment:
We could just make this `move_to_start()` and get rid of the `last` option (since that's all we will ever use), but I went with this for now to stay consistent with the `OrderedDict` call